src/components/quote/README.md

```diff
@@ -0,0 +1,47 @@
+# Quote
+Call prominent attention to a quotation using large text and a citation
+
+## When to use
+Use the quote component when you have a quotation that you would like to display prominently in your page. This could include:
+
+- A sentence or short phrase quoted by an important person
+- A short testimonial from a user about a specific product
+
+## When to use something else
+If the quote is too long to easily read, consider using the HTML `blockquote` element or HTML paragraph styles instead.
+
+## Adding the markup
+
+The container for a quote is made up of a `div` with a class of `rvt-quote`.
+
+```
+<div class="rvt-quote">
+
+</div>
+```
+
+The visual display of the quote is made up of a `blockquote` element with a class of `rvt-quote__text`, a `p` element with the quote's text, and a `cite` element with the author's name.
```

Comment:
The phrase "author's name" implies that the quote was originally in written form. Consider rephrasing to something like 'element with the attribution`, 'element with the attributor's name', etc.